webapp/test/datastore_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:convert' show json;
+
+import 'package:test/test.dart';
+
+import 'package:CodaV2/datastore.dart' as ds;
```

Comment:
s/datastore/data_model/

So we don't confuse it with Cloud Datastore

Reply:
Done.